### DISCUSSION TITLE
A test notification after initial setup

### DISCUSSION THREAD

**Discussion by Gregwolf3030** [martomi/chiadog#127] [Ideas]
It would be a nice feature that the system would send an email (or whatever notification you setup) upon initial load of the script. That way the user would know that the notification function is working and the system is running. Currently, after you start it up for the first time, you don't know. You just have to wait until either an event gets triggered or you get your first report.

**Comment by ghost** [comment]
is there a way to manually test this? I have set up the smtp settings, I know they work. I can telnet to the relay server, my credentials are correct, but not notifications. I rename 4 plots, I see the warning at the command prompt, but no notification.

**Reply by martomi** [reply]
Yes, you can manually test by running the corresponding unit test as described [here](https://github.com/martomi/chiadog/blob/main/INTEGRATIONS.md).
